### DISCUSSION TITLE
Second Community Meeting 26/05/2022

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#6675] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfill this, we would like to held every Month Community Meetings where we can share main updates and also have an open discussion with you.

---

**💥Second Community Meeting will take place on Thursday 26/05/2022 at 4 pm GMT+2 time zone. (1 hour length) We will use Google Meet.**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you participated on the First Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you joins us again or not.

---

### 1) Espressif Introduction 
Meeting purpose & rules
### 2)Development updates
Summary of our current development and news
### 2) Community topics
- Implementation of Command Console ​
- Add support for esp_sr WakeNet
- Add support for ATEC608A​
### 3) Open Discussions
Place for any topic which is not in Agenda.

See you on the Meeting!

**Comment by VojtechBartoska** [comment]
Hello all,

**one of the topic we would like to open is sorting **Feature requests** by Community requests.**

Is there any feature you would like to implement? If yes, please share it here in comments and we can discuss it more on the Meeting.

List of current features which are added to the Roadmap are available [here](https://github.com/orgs/espressif/projects/3/views/7).

**Comment by mrengineer7777** [comment]
As far as adding new features it would be cool to have a command console.  I know that's available in ESP-IDF but not sure how it would integrate with Arduino.  Or perhaps there's a simple way to read the serial data and implement a basic console?  I think that's what Tasmota does.

https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/console.html

**Reply by me-no-dev** [reply]
Will add console support. Have been on my mind for some time :)

**Reply by VojtechBartoska** [reply]
@mrengineer7777 Adding this to the Agenda.

**Comment by torntrousers** [comment]
How hard would it be for ESP32/Arduino to support [esp_sr WakeNet](https://github.com/espressif/esp-sr/tree/37104cb56b36db2de80e0604cb6f9f07236bae84/docs/wake_word_engine)?  All sorts of fun DIY things you could make triggered with a few wake words.

**Reply by me-no-dev** [reply]
already supported for S3. Will add an example soon :) only Chinese available for the other chips, so they do not have the feature enabled for now.

**Reply by torntrousers** [reply]
I'll have to go buy an S3...

**Reply by torntrousers** [reply]
Ordered an [S3-Box](https://www.espressif.com/en/news/ESP32-S3-BOX_video) and its arrived now so looking forward to that example...

**Comment by torntrousers** [comment]
Another thing that would be interesting for me would be if ESP32/Arduino could support the ATECC608A like the IDF does - https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/protocols/esp_tls.html#atecc608a-secure-element-with-esp-tls. Any chance of that?

**Comment by VojtechBartoska** [comment]
Agenda updated up to your proposals. Thanks everyone for nice ideas!

**Comment by mrengineer7777** [comment]
Good to meet the fellow developers I'm working with on this framework.  Since Microframe is switching to ESP32 for our WiFi connectivity, I plan to stay active in the community.  Looking forward to seeing this framework continue to improve and grow.

**Comment by VojtechBartoska** [comment]
Sharing the slides from the Meeting:
[Second_Community_meeting-Arduino-May_2022.pdf](https://github.com/espressif/arduino-esp32/files/8779756/Second_Community_meeting-Arduino-May_2022.pdf)
